README.md

```diff
@@ -1,8 +1,45 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Hunain Khalid . Jarvis Consulting
 
+I am a recent graduate from York University's Computer Science program. I specialize in software development, from compiler-level code to machine learning. Although proficient, I continue to improve my skills in Docker, Java, SQL, Python, React.js, Git, Linux and more. As a full-stack software developer, I solve problems quickly, with optimized run time and extensive documentation. A course on computer security helped with enterprise-level solutions to computer security and security at the programming level. I am also an excellent communicator, team member and leader. Currently employed at Jarvis, I improve my skills through difficult projects and team collaboration in an agile environment.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git/Github
+
+**Competent:** Docker, ReactJS/GatsbyJS/NextJS, AngularJS, Python, Android
+
+**Familiar:** C programming language, Numpy, UX/UI design, Advanced Object Oriented Programming, Kotlin, Pandas
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid](https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid/tree/master/linux_sql)]: Developed a linux cluster monitoring agent for the Jarvis Cluster Team (JCT).The application sends reports of server usage to the JCT. Reports determine whether to add or remove servers. Implemented in CentOS7, using a Docker container with an attached PSQL instance and IntelliJ's bash script extension. Scripts and SQL code are sent to Jarvis remote servers. Once sent, monitoring of database insertions was done in real-time through database viewer, DBeaver. Testing was done manually. All code is deployed to the git url above.
```

Comment:
**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid/tree/master/linux_sql)]: Developed a Linux cluster monitoring agent to monitor and analyze server hardware usages in real-time using Bash script, PostgreSQL, and Docker. Bash scripts .... psql. Data analytics and reporting with SQL. Docker ... . crontab git.